Find a functionally equivalent code:com.android.internal.os.RuntimeInit.zygoteInit:COMMENT
Method Modifier: public      static      final       internal    
Comment:/**
 * The main function called when started through the zygote process. This
 * could be unified with main(), if the native code in nativeFinishInit()
 * were rationalized with Zygote startup.<p>
 *
 * Current recognized args:
 * <ul>
 * <li> <code> [--] &lt;start class name&gt;  &lt;args&gt;
 * </ul>
 *
 * @param targetSdkVersion target SDK version
 * @param argv arg strings
 */

@@ -4,8 +4,8 @@
     redirectLogStreams();
     commonInit();
     nativeZygoteInit();
-    applicationInit(targetSdkVersion, argv);
+    applicationInit(targetSdkVersion, argv, classLoader);
 }

